docs/Api/DefaultApi.md

```diff
@@ -1417,6 +1423,376 @@ Name | Type | Description  | Notes
 [[Back to Model list]](../../README.md#models)
 [[Back to README]](../../README.md)
 
+## `getTenDLCSmsBrand()`
+
+```php
+getTenDLCSmsBrand($account_id, $brand_id): \FreeClimb\Api\Model\SMSTenDLCBrand
+```
+
+Get a 10DLC SMS Brand
+
+### Example
+
+```php
+<?php
+require_once(__DIR__ . '/vendor/autoload.php');
+
+
+// Configure HTTP basic authorization: fc
+$config = FreeClimb\Api\Configuration::getDefaultConfiguration()
+              ->setUsername('YOUR_ACCOUNT_ID')
+              ->setPassword('YOUR_API_KEY');
+
+
+$apiInstance = new FreeClimb\Api\Api\DefaultApi(
+    // If you want use custom http client, pass your client which implements `GuzzleHttp\ClientInterface`.
+    // This is optional, `GuzzleHttp\Client` will be used as default.
+    new GuzzleHttp\Client(),
+    $config
+);
+$account_id = 'account_id_example'; // string | ID of the account
+$brand_id = 'brand_id_example'; // string | String that uniquely identifies this brand resource.
+
+try {
+    $result = $apiInstance->getTenDLCSmsBrand($account_id, $brand_id);
```

Comment:
This documentation is not accurate is it? do we have to manually supply the account id? or will the SDK handle that for us?

Reply:
Yeah it's not accurate, the SDK will handle that for us, the tests don't supply an account id either, will make this change